src/Umbraco.Web.UI.Client/src/packages/tags/components/tags-input/tags-input.element.ts

```diff
@@ -78,13 +78,23 @@ export class UmbTagsInputElement extends UUIFormControlMixin(UmbLitElement, '')
 		this._matches = data.items;
 	}
 
-	#onKeydown(e: KeyboardEvent) {
+	#onKeydown(e: KeyboardEvent, idx: number) {
 		//Prevent tab away if there is a input.
 		if (e.key === 'Tab' && (this._tagInput.value as string).trim().length && !this._matches.length) {
 			e.preventDefault();
 			this.#createTag();
 			return;
 		}
+
+		//To be able to get out of the input
+		if (e.key === 'Tab' && !(this._tagInput.value as string).trim().length) return;
```

Comment:
For readability I would like us to always use { } for if-sentences, we only skip it if it is a very simple if-sentence :-)